hax/hax/filestats.py

```diff
@@ -57,6 +57,10 @@ def _execute(self, motr: Motr):
             ffi.adopt_motr_thread()
             self._ensure_motr_all_started()
             while not self.stopped:
+                if not self._am_i_rc():
```

Comment:
[Style] suggestion: change "_am_i_rc()" to shorter and more readable variant: "_is_rc()".

Reply:
Not sure it is self descriptive "Is RC?" - "Who are you talking about?"

Reply:
@chumakd What's your opinion? I would love to get this PR merged, so I'd like to resolve all the discussions :-)